modules/ROOT/pages/mule-app-dev.adoc

```diff
@@ -0,0 +1,71 @@
+= Mule Application Development
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Mule applications enable you to perform data integrations for your use cases.
+Typical Mule applications read data from internal and external sources,
+process and transform data to the required formats and structures, and
+write that output to the systems and servers where you store and use the
+transformed data.
+
+//TITLE TBD:
+== Mule Applications
+
+You can start a Mule app with or without an API specification, connect to
+external data sources using connectors, process your data with Core components,
+and create other configurations you need to complete your application.
+
+* xref:mule-application-about.adoc[Mule Apps]:
+Learn basic Mule concepts.
+* xref:about-mule-configuration.adoc[Mule Configuration]: Understand the XML
+structure and configurations of Mule apps, domains used for sharing resources
```

Comment:
I would not mention domains yet to keep it simple, nor policies. Maybe there should be a parent section that provides a high level description of each artifact and then the user can dive deeper into each of them starting from Mule apps?